src/tup/path.c

```diff
@@ -124,7 +124,7 @@ static int watch_path_internal(tupid_t dt, const char *file,
 			if(subtent) {
 				tupid_tree_remove(&root, subtent->tnode.tupid);
 			}
-			if(f.filename[0] == '.') {
+			if(f.filename[0] == '.' || f.filename[0] == '_') {
```

Comment:
I believe the original reason for skipping paths starting with '.' here was just based on an idea that build inputs should only include files that aren't hidden by default in the shell or file browser (on Linux). Do underscore-prefixed filenames have similar special meaning somewhere?

I see you ignore "_darcs" elsewhere, which is fine, but I'm wondering if that's sufficient without this change (and the similar one in pel_group.c)